DOCUMENTATION.md

```diff
@@ -217,7 +227,7 @@ client.delete_device(dev_id)
 ```
 - `dev_id`: **string**  the id of the device to be deleted.
 
-### Device
+### DeviceObject
```

Comment:
I had to rename it because of the device method which created problems when linking to the section of the Device object
I'll put the old name back and see if there is still conflicts

Reply:
You can add new anchors in Markdown documents with the HTML `<a>` tag:

```markdown
# Part 1

[Some link](#anch)

# <a name="anch"></a>Part 2

Some text
```

In this case, clicking on "Some link" will redirect to Part 2.